Inquisitor.c

```diff
@@ -1,56 +1,60 @@
-//Inquisitor (Parsec Support tool)
+// Inquisitor (Parsec Support tool)
 
-#include <stdio.h> //File IO
-#include <stdlib.h> //Used for the Dynamic File Path
+#include <stdio.h> // File IO
+#include <stdlib.h> // Used for the Dynamic File Path
 #include <string.h> 
 
 
-//Gets & prints out the Parsec Log
-int getLog(){
-    //Open Log File for Reading
-    FILE* fp = fopen(strcat(getenv("USERPROFILE"),"/AppData/Roaming/Parsec/log.txt"),"r");
+/ /Gets & prints out the Parsec Log
+static void getLog(const char *path)

```

Comment:
Changing the function to a void makes the return at line 18 break. A static int would work